cypress/e2e/api/engageSphere.cy.js

```diff
@@ -0,0 +1,135 @@
+describe('EngageSphere API - Customers', () => {
+  it('Recupera clientes com sucesso (por exemplo, verifica o código de status 200)', () => {
+    cy.request(`${Cypress.env("apiUrl")}/customers`).as('getCustomers');
+
+    cy.get('@getCustomers').should((response) => {
+      expect(response).to.have.property('status', 200);
+    });
+  });
+
+  it('Realiza a paginação de clientes corretamente', () => {
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: {
+        page: 2,
+        limit: 1,
+        size: 'All',
+        industry: 'All'
+      }
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(response.body).to.have.property('pageInfo');
+      expect(response.body.pageInfo).to.deep.include({
+        currentPage: 2,
+      });
+    });
+  });
+
+  it('Filtra corretamente o limite de clientes', () => {
+    const limit = 2;
+
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
+      qs: {
+        page: 2,
+        limit: limit,
+        size: 'All',
+        industry: 'All'
+      }
+    }).then((response) => {
+      const totalCustomers = response.body.pageInfo.totalCustomers;
+      const expectedTotalPages = Math.ceil(totalCustomers / limit);
+
+      expect(response.body.pageInfo).to.deep.include({
+        totalPages: expectedTotalPages,
+        totalCustomers: totalCustomers
+      });
+    });
+  });
+
+  it('Trata requisições inválidas (por exemplo, página negativa)', () => {
+    cy.request({
+      method: 'GET',
+      url: `${Cypress.env("apiUrl")}/customers`,
```

Comment:
Visto que esta URL se repete em vários testes, que tal criar uma variável chamada `CUSTOMERS_API_URL` no início do arquivo, a qual poderia ser reaproveitada ao longo de todos os testes sem repetir `${Cypress.env("apiUrl")}/customers`?